specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/settings.json

```diff
@@ -0,0 +1,137 @@
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage.Admin/locations/{location}/settings": {
+      "post": {
+        "x-ms-examples": {
+          "Update srp settings.": {
+            "$ref": "./examples/Settings/Update.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Update storage resource provider settings.",
+        "operationId": "StorageSettings_Update",
+        "parameters": [
+          {
+            "$ref": "storage.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "storage.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "storage.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "Parameters used to update storage resource provider settings",
+            "name": "settingsObject",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/settings"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The new storage resource provider setting is updated.",
+            "schema": {
+              "$ref": "#/definitions/settings"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "storage.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "x-ms-examples": {
+          "Returns the storage resource provider settings.": {
+            "$ref": "./examples/Settings/Get.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Returns the storage resource provider settings.",
+        "operationId": "StorageSettings_Get",
+        "parameters": [
+          {
+            "$ref": "storage.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "storage.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "storage.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The settings has been returned.",
+            "schema": {
+              "$ref": "#/definitions/settings"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "storage.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "settings": {
+      "description": "Storage resource provider settings",
+      "type": "object",
+      "properties": {
+        "retentionPeriodForDeletedStorageAccountsInDays": {
+          "description": "Set the retention days for deleted storage accounts.",
```

Comment:
What does this mean when you do a GET?  You aren't setting anything,

You should makes comments as useful as possible to the end user; something like this would be much better
```
The number of days a deleted storage account is kept before being permanently deleted.
```

Reply:
fixed